edx_proctoring/backends/backend.py

```diff
@@ -63,6 +65,22 @@ def get_javascript(self):
         """
         return ""
 
+    def get_ping_failure_should_block_user(self):
```

Comment:
This will probably need to be determined per attempt, not per backend. If it's set globally like this, then the ping is irrelevant. 

The poll interval can be set per backend, but the failure handling could be implemented using `ProctoredExamStudentAllowance`.

Reply:
This reminded me to update the two tickets we had subsumed into this work to reference one another; this is out of scope.